ports/wordswithfreds/Words With Freds.sh

```diff
@@ -0,0 +1,60 @@
+#!/bin/bash
+
```

Comment:
Old header, please use the new header.
https://portmaster.games/packaging.html

Reply:
Okies, Updated, please review